test/AlertBox_test.jsx

```diff
@@ -1,38 +1,41 @@
 import _ from "lodash";
 import assert from "assert";
 import React from "react";
-import { shallow } from "enzyme";
+import { render, cleanup } from "@testing-library/react";
 import { AlertBox } from "../src";
 
+afterEach(cleanup);
+
 describe("AlertBox", () => {
   _.forEach(["warning", "info", "error", "success"], type => {
     it(`renders a <AlertBox> element with class AlertBox--${type}`, () => {
-      const alertBox = shallow(
+      const { getByTestId } = render(
         <AlertBox type={type} title="title">
           <p> child </p>
         </AlertBox>,
       );
-      assert.equal(alertBox.find(`.AlertBox--${type}`).length, 1);
+      console.log(getByTestId("AlertBoxType").classList);
+      assert.ok(getByTestId("AlertBoxType").classList.contains(`AlertBox--${type}`));
     });
   });
 
   it("renders a closable box if isClosable and closes on click", () => {
-    const alertBox = shallow(
+    const { getByTestId, container } = render(
       <AlertBox type="warning" title="title" isClosable>
         <p> child </p>
       </AlertBox>,
     );
-    assert.equal(alertBox.find(".AlertBox--warning").length, 1);
-    alertBox.find(".AlertBox--close").simulate("click");
-    assert.equal(alertBox.find(".AlertBox--warning").length, 0);
+    assert.ok(container.firstChild);
+    getByTestId("AlertBoxCloseButton").click();
```

Comment:
I think you can find the close button with `getByLabelText`, since it has an ARIA label.